packages/nimble-components/src/accordion/specs/README.md

```diff
@@ -0,0 +1,144 @@
+
+---
+
+## Design
+
+Simple accordion group
+
+```
+<fast-accordion>
+    <fast-accordion-item>
+        <span slot="heading">Panel one</span>
+        <span slot="icon">^</span>
+        Panel one content
+    </fast-accordion-item>
+    <fast-accordion-item expanded>
+        <span slot="heading">Panel two</span>
+        <span slot="icon">^</span>
+        Panel two content
+    </fast-accordion-item>
+    <fast-accordion-item>
+        <span slot="heading">Panel three</span>
+        <span slot="icon">^</span>
+        Panel three content
+    </fast-accordion-item>
+</fast-accordion>
+```
+
+Complex accordion group with components in slots and single expand-mode
+
+```
+<fast-accordion expand-mode="single">
+    <fast-accordion-item>
+        <fast-checkbox slot="start"></fast-checkbox>
+            <fast-select slot="end">
+                <fast-option value="1">Option 1</fast-option>
+                <fast-option value="2">Option 2</fast-option>
+                <fast-option value="3">Option 3</fast-option>
+            </fast-select>
+                Accordion one content
+            <div slot="heading">Accordion one</div>
+    </fast-accordion-item>
+    <fast-accordion-item>
+        <fast-checkbox slot="start"></fast-checkbox>
+            <fast-number-field placeholder="number" slot="end"></fast-number-field>
+                Accordion two content
+            <div slot="heading">Accordion two</div>
+    </fast-accordion-item>
+    <fast-accordion-item>
+        <fast-checkbox slot="start"></fast-checkbox>
+            <fast-select slot="end">
+                <fast-option value="1">Option 1</fast-option>
+                <fast-option value="2">Option 2</fast-option>
+                <fast-option value="3">Option 3</fast-option>
+            </fast-select>
+            Accordion three content<div slot="heading">Accordion three</div>
+    </fast-accordion-item>
+    <fast-accordion-item>
+        <fast-checkbox slot="start"></fast-checkbox>
+            <fast-number-field placeholder="number" slot="end"></fast-number-field>
+                Accordion four content
+            <div slot="heading">Accordion four</div>
+    </fast-accordion-item>
+</fast-accordion>
+```
+
+### API
+
+#### FAST Accordion
+
+[FAST accordion API](https://github.com/microsoft/fast/blob/57f3c22c6341d8a21d48b1ffb7fcbfab1ffd02d8/packages/web-components/fast-foundation/src/accordion/accordion.spec.md)
+
+-   _Component Name:_ `nimble-accordion`
+-   _Properties/Attributes:_ Unchanged
+-   _Methods:_ Unchanged
+-   _Events:_ Unchanged
+-   _CSS Classes and Custom Properties that affect the component:_ Unchanged
+-   _Slots:_ Unchanged
+
+#### FAST Accordion Item
+
+[FAST accordion-item API](https://github.com/microsoft/fast/tree/57f3c22c6341d8a21d48b1ffb7fcbfab1ffd02d8/packages/web-components/fast-foundation/src/accordion-item)
+
+-   _Component Name:_ `nimble-accordion-item`
+-   _Properties/Attributes:_
+    -   `appearance` - `block`, `outline`, `ghost`
```

Comment:
Is there a reason to explicitly provide `appearance` on the items themselves? Why not provide it on the `nimble-accordion` component? I wouldn't expect that we want items to have different appearances within a single accordion component.

Reply:
Yeah, that's a good point. I'll move those attributes to the nimble-accordion component's `Properties/Attributes` section 👍